syncopy/tests/test_specest.py

```diff
@@ -918,6 +918,7 @@ class TestWavelet():
                        "channels": range(0, int(nChannels / 2)),
                        "toilim": [-20, 60.8]}]
 
+    @skip_low_mem
```

Comment:
Uh this magic is new to me :)

Reply:
:D Let's see if that helps w/testing pipeline stability on low-spec systems. Fingers crossed!